Sources/NIOPosix/Bootstrap.swift

```diff
@@ -712,9 +712,14 @@ private extension Channel {
 
 /// A `ClientBootstrap` is an easy way to bootstrap a `SocketChannel` when creating network clients.
 ///
-/// Usually you re-use a `ClientBootstrap` once you set it up and called `connect` multiple times on it.
+/// You may re-use a `ClientBootstrap` once you set it up and called `connect` multiple times on it.
 /// This way you ensure that the same `EventLoop`s will be shared across all your connections.
 ///
+/// Keep in mind that `ClientBoostrap` is not `Sendable` so you cannot share the same instance across
```

Comment:
```
Keep in mind that `ClientBootstrap` is not ``Sendable``, 
so you cannot share the same instance across multiple 
``EventLoop``s or multiple concurrency domains in general. 
Creating a `ClientBootstrap` is cheap, so instead of 
arranging concurrent code to re-use a single 
`ClientBootstrap` instance across threads, it is often 
easier to create a dedicated instance for each task.
```

Reply:
```suggestion
/// Keep in mind that `ClientBoostrap` is not ``Sendable``, so you cannot share the same
/// instance across multiple ``EventLoop``s or multiple concurrency domains in general. 
/// Creating a `ClientBootstrap` is cheap. So instead of arranging synchronization allowing
/// concurrent code to re-use a single `ClientBootstrap` instance across many tasks, it is often 
/// easier to create a dedicated instance for each task.
```